public/js/components/FormFields/FormFieldDateTextInput.js

```diff
@@ -0,0 +1,58 @@
+import React, {PropTypes} from 'react';
+import ShowErrors from '../Utilities/ShowErrors';
+import { errorPropType } from '../../constants/errorPropType';
+import moment from 'moment';
+
+export default class FormFieldDateTextInput extends React.Component {
+
+
+    static propTypes = {
+        fieldLabel: PropTypes.string,
+        fieldName: PropTypes.string,
+        fieldValue: PropTypes.number,
+        fieldErrors: PropTypes.arrayOf(errorPropType),
+        formRowClass: PropTypes.string,
+        onUpdateField: PropTypes.func
+    };
+
+    milisecondsToString = (ms) => {
```

Comment:
this is a static method, you can move it out of the class and get rid of the annoying `this` juggling.